lib/improver/tests/wind_gust_diagnostic/test_WindGustDiagnostic.py

```diff
@@ -208,7 +208,7 @@ def test_fails_if_data_is_not_cube(self):
                                            "wind_speed_of_gust")
 
     def test_fails_if_no_perc_coord(self):
-        """Test it raises a Value Error if there is no percentile coord."""
+        """Test it raises an Error if there is no percentile coord."""
```

Comment:
Should any of these tests be removed from this file as they are now in `test_cube_checker.py` instead, or do you think we should keep these tests in here anyway?

Reply:
I think codacy is now complaining that these tests are in two places, so we should probably remove them from here.

Reply:
This is a bit of a difficult one. We are factoring out a bit of code so the tests in the original unit test are now required for the new bit of code. However we want to be sure that the functionality of the original code hasn't changed i.e. it will still fall over in the same place as before even if the Error is now raised from the new bit of code. The unit tests have all succeeded so I suppose now we can delete them and reduce the complexity of the code (which is what codacy is warning about) but I don't think having too many tests is an error and even codacy isn't flagging it as such.